src/providers/boleto-api-caixa/index.js

```diff
@@ -120,10 +122,15 @@ const translateResponseCode = (axiosResponse) => {
   const hasErrors = path(['errors', 'length'], axiosResponseData)
 
   if (!hasErrors) {
+    const boletoResponseLinks = path(['links'], axiosResponseData)
+    const boletoHtml = find(whereEq({ rel: 'html' }))(boletoResponseLinks)
+    const boletoUrl = boletoHtml.href
```

Comment:
Apenas uma sugestão do deus do Ramda!
```suggestion
const isHtml = propEq('rel', 'html')

const getBoletoHtmlLink = pipe(
  path(['links']),
  find(isHtml),
  prop('href')
)(axiosResponseData)
```

Reply:
:fire: :1st_place_medal: :guitar:

Reply:
não vou usar só pq se se chamou de Deus

Reply:
```suggestion
const isHtml = propEq('rel', 'html')

const getBoletoHtmlLink = pipe(
  path(['links']),
   ifElse(
     isHtml,
     prop('href'),
    always('other_format')
    )
)(axiosResponseData)
```

Reply:
é guerraaaaa

Reply:
```
const { links: boletoResponseLinks } = axiosResponseData || {}
const { boletoUrl: href} = boletoResponseLinks.find(({ rel }) => rel === 'html')
```

Reply:
Me bassei na do J e fiz outra hehehehhe :dagger: